chandra_aca/tests/test_maude_decom.py

```diff
@@ -14,6 +15,46 @@
     test_data.update(pickle.load(f))
 
 
+def compare_tables(t1, t2, keys=None, exclude=()):
+    assert len(t1) == len(t2)
+    assert sorted(t1.colnames) == sorted(t2.colnames)
+    _compare_common_columns(t1, t2, keys, exclude)
+
+
+def _compare_common_columns(t1, t2, keys=None, exclude=()):
+    from astropy import table
+
+    if keys is None:
+        keys = [k for k in t1.colnames if k in t2.colnames]
+    for key in exclude:
+        keys.remove(key)
+    errors = []
+    for name in keys:
+        col_1, col_2 = t1[name], t2[name]
+        ok = type(col_1) == type(col_2)
+        if type(col_1) is table.MaskedColumn and type(col_1) is table.MaskedColumn:
+            ok *= np.all(col_1.mask == col_2.mask)
+        if (np.issubdtype(col_1.dtype, np.inexact) or
+                np.issubdtype(col_1.dtype.base, np.inexact)):
+            c = (np.isclose(col_1, col_2) | (np.isnan(col_1) & np.isnan(col_1)))
+        else:
+            c = (col_1 == col_2)
+        if type(col_1) is table.MaskedColumn:
+            if np.sum(~c.mask):
```

Comment:
This would normally be written `np.any(~c.mask)`. Again, using logical operators instead of the implicit int version of bools. `np.any` has the advantage that it will stop looking once the first `True` is found.

Reply:
Fixed